### PR TITLE
Testing website

### PR DESCRIPTION
**Pull request by elberthcorniell**
In this pull request we want to add testings for:

- Routes
- Home
- Quotes
- Calculator

### CODE REVIEW

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @elberthcorniell 

Great job with the unit testing. It's really world-class work from you
Screen Shot 2021-01-23 at 3 27 27 AM

I will need you you generate snapshots instead of rendering from the test and then checking some strings that appear on the DOM.
Please generate me some snapshots.
Waiting for a final review with some adjustments
Happy coding

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @elberthcorniell, congratulations on a great job done :smile: Having added the snapshot test as outlined by the previous TSE :+1: I hereby approve the project :+1: 

Kindly go ahead and merge into development, and continue with your next projects :slightly_smiling_face: 
Do not forget to submit the project completion form :1st_place_medal: 
_Happy coding_ :woman_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**
